api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -630,6 +630,11 @@ message ResponseMapper {
   // or to override headers of any local reply before it is sent to a downstream client.
   repeated config.core.v3.HeaderValueOption headers_to_add = 5
       [(validate.rules).repeated = {max_items: 1000}];
+
+  // Tokenized HTTP headers to add to a local reply. This allows the response mapper to append, to add
+  // or to override headers of any local reply before it is sent to a downstream client.
+  repeated config.core.v3.TokenizedHeaderValueOption tokenized_headers_to_add = 6
```

Comment:
I don't fully grok how this works as an end-user. Can you add some docs with a complete example? I.e. how is this any more useful than regular `headers_to_add`, why doesn't that work with local reply mapper, etc?

Reply:
In addition to the question above: do we actually want to allow using the header formatter? Wouldn't that give us a fully generic way to get what we want?